File: csharp/RequestReplay/MyLog.cs
using System.Collections.Generic;

public class MyLog
{
  public MyRequest request { get; set; }
  public MyResponse response { get; set; }
}

public class MyResponse
{
  public string body { get; set; }
  public int code { get; set; }
}

public class MyRequest
{
  public string url { get; set; }
  public string method { get; set; }
  public Dictionary<string, string> headers { get; set; }
  public string body { get; set; }
}

File: kotlin/src/main/kotlin/com/stripe/interview/MyLog.kt
package com.stripe.interview

data class MyResponse(
    val body: String,
    val code: Long

)

data class MyRequest(
    val url: String,
    val header: Map<String, String>,
    val body: String,
    val method: String
)

data class MyLog(
    val request: MyRequest,
    val response: MyResponse
)